src/Algebra/Morphism/RingMonomorphism.agda

```diff
@@ -127,3 +127,9 @@ isRing isRing = record
   ; distrib          = distrib R.+-isGroup R.*-isMagma R.distrib
   ; zero             = zero R.+-isGroup R.*-isMagma R.zero
   } where module R = IsRing isRing
+
+isCommutativeRing : IsCommutativeRing _≈₂_ _⊕_ _⊛_ ⊝_ 0#₂ 1#₂ → IsCommutativeRing _≈₁_ _+_ _*_ -_ 0# 1#
```

Comment:
At this point we can split the type signature on two lines, I think.

Reply:
Done and fixup'd into the original commit.